vendor/llm2vec_monGARS/llm2vec/dataset/utils.py

```
@@ -0,0 +1,24 @@
+    Args:
+        dataset_name (str): Name of the dataset to load.
+        split (str): Split of the dataset to load.
+        file_path (str): Path to the dataset file.
+    """
+    dataset_mapping = {
+        "E5": E5Data,
+        "Wiki1M": Wiki1M,
+    }
+
+    if dataset_name not in dataset_mapping:
+        raise ValueError(f"Dataset name {dataset_name} not supported.")
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)